_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -477,6 +490,32 @@ public static function get_plans( $request ) {
 		return $data;
 	}
 
+	public static function get_checklist() {
+		$site_id = Jetpack_Options::get_option( 'id' );
+		$response = Jetpack_Client::wpcom_json_api_request_as_user(
+			'sites/' . $site_id . '/checklist',
+			'1',
+			array(
+				'method'  => 'GET',
+				'headers' => array(
+					'X-Forwarded-For' => Jetpack::current_user_ip( true ),
```

Comment:
Just curious; is this header needed? I see other functions in this file have it but e.g. this doesn't:

https://github.com/Automattic/jetpack/blob/8f05c29dabbfe7161a3f932097bc1663db78d035/_inc/lib/core-api/wpcom-endpoints/memberships.php#L111-L123

Reply:
I'm not sure, this was just working from similar request.

Data bits have moved to a follow-up PR: https://github.com/Automattic/jetpack/pull/12404